### ISSUE TITLE
Add  Arithmetic Question in reviews, comments, register & log in forms.

### ISSUE THREAD

**Issue by davidovic999** [bhadaway/stop-spammers#141]
Hi,

I need to know if the plugin adds Arithmetic Question in: reviews & comments forms, my account register & log in forms.

I checked Arithmetic Question in “Second Chance CAPTCHA Challenge”, but no Arithmetic Question displayed on the forms.

How to implement this correctly?

Thank you.

**Comment by bhadaway** [comment]
No, this is for when users are caught in the spam filter and blocked, it gives them a second chance to prove that they're not a bot. This is a good idea though, to be able to place the CAPTCHAs on front-end forms, so I'll mark it as a suggestion for possible future implementation.

In the meantime, check out this plugin to add the additional protection:

https://wordpress.org/plugins/invisible-recaptcha/
